docs/data/external-oasis-repos.md

```diff
@@ -1,60 +1,79 @@
+OASIS (the **OSG** **A**pplication **S**oftware **I**nstallation **S**ervice) is an infrastructure, based on
+[CVMFS](https://cvmfs.readthedocs.io), for distributing software throughout the OSG.  Once software is installed into
+an OASIS repository, the goal is to make it available across about 90% of the OSG within an hour.
 
+OASIS consists of keysigning infrastructure, a content distribution network (CDN), and a shared CVMFS repository that
+is hosted by the OSG.  Many use cases will be covered by utilizing the [shared repository](update-oasis); this document
+covers how to install, configure, and host your own CVMFS _repository server_.  This server will distribute software
+via OASIS, but be hosted and operated externally from the OSG project.
 
-About OASIS
------------
+OASIS-based distribution and key signing is available to OSG VOs or repositories affiliated with an OSG VO.
+See the [policy page](https://opensciencegrid.github.io/technology/policy/external-oasis-repos/) for more information
+on what repositories OSG is willing to distribute.
 
-OASIS is the OSG Application Software Installation Service. It is the recommended method to install software on the Open Science Grid. It is implemented using CernVM FileSystem (CVMFS) technology.
+Before Starting
+---------------
 
-About this Document
--------------------
+CVMFS repositories work at the kernel filesystem layer, which adds more stringent host requirements than a typical
+OSG install.  The host OS must meet ONE of the following:
 
-This document describes the detailed procedures for dealing with CVMFS repositories that are part of the OASIS system but are not hosted at the OSG Global Operations Center (GOC). Rather, they are hosted at the home institution of a Virtual Organization (VO). The procedures include the steps performed by both the GOC and by the repository service administrator. If you are only interested in using an OASIS repository that is hosted at the GOC, see [this document](update-oasis) instead.
+-   RHEL 7.3 (or equivalent) or later.  **This option is recommended**.
+-   RHEL 6 with the aufs kernel module.
 
-Policies
---------
+Additionally,
 
-These are the policies regarding OSG CVMFS repositories:
+-   **User IDs:** If it does not exist already, the installation will create the `cvmfs` Linux user
+-   **Group IDs:** If they do not exist already, the installation will create the Linux groups `cvmfs` and `fuse`
+-   **Network ports:** This page will configure the repository to distribute using Apache HTTPD on port 8000.  At the
+    minimum, the repository needs in-bound access from the OASIS CDN.
+-   **Disk space:** This host will need enough free disk space to host _two_ copies of the software: one compressed
+    and one uncompressed. `/srv/cvmfs` will hold all the published data, while `/var/spool/cvmfs` will contain all
+    the data in the current transaction.
```

Comment:
change "all the publishe data" to "all the published data in compressed and de-duplicated form" and change "current transaction" to "current transaction in uncompressed form.  `/var/spool/cvmfs` should contain enough space for the largest amount of data expected to be published in a transaction, and possibly more than one in parallel if the server hosts more than one repository."

Reply:
I updated the text, but trimmed down the wording a bit (for example, I think it's implicit that these should be sized according to the largest possible transaction...).